documentation/docs/steps/abapEnvironmentCreateSystem.md

```diff
@@ -68,3 +68,64 @@ create-services:
   plan:   "standard"
   parameters: "{ \"admin_email\" : \"user@example.com\", \"description\" : \"ABAP Environment Q System\", \"is_development_allowed\" : true, \"sapsystemname\" : \"H02\", \"size_of_persistence\" : 4, \"size_of_runtime\" : 1 }"
 ```
+
+## Example: Configuration of an abap-oem Service
+
+If you want to use an abap-oem System please note that the following configurations need to be specified correctly:
+
+* `abapSystemParentSaasAppname`: needs to fulfill the syntax [a-zA-Z0-9\-\_]+
+* `abapSystemConsumerTenantLimit`: The number of consumer tenants tmust be greater or equal 1
+* Either `abapSystemParentServiceLabel` or `abapSystemParentSaasAppname` must be set - depending on who created the oem-instance
```

Comment:
depending on who created the oem-instance
-> I do not understand this part. Is it possible to be more specific?

Reply:
If we need more description here, then we need to involve Welf. abapSystemParentServiceLabel does not seem to be used for now. abapSystemParentSaasAppname shall be the appName provided as parameter for the SaaS Provisioning service (saas-registry).

Reply:
Can you provide more information @peterpersiel ? I simply added the information from https://github.wdf.sap.corp/Steampunk/Documentation/wiki/Service-Documentation#parameters-for-oem. This only states: Either parent_service_label or parent_saas_appname must be set - depending on who created the oem-instance

Reply:
It think it is sufficient to only explain that one of both needs to be set. Only abapSystemParentSaaSAppname (https://github.wdf.sap.corp/Steampunk/Documentation/wiki/Service-Documentation#parent_saas_appname) will be mentioned in the documentation for the ABAP OEM Service (tbd: https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/1697387c02e74e66a55cf21a05678167.html). For the other parameters there is no concept yet how these will be used (service instead of SaaS on top of abap-oem).

Reply:
Thank you for the clarification. So I suggest to keep current explanations and to extend them further if necessary?